Automattic/jetpack
# Subscription Block: Add newsletter categories to the content view

Closes https://github.com/Automattic/wp-calypso/issues/80905
Depends on https://github.com/Automattic/jetpack/pull/32666

## Proposed changes:

* Add newsletter categories to the Subscription Block content view
* Make the border-radius option of the Subscribe block affect the category pills on the content view https://github.com/Automattic/wp-calypso/issues/80908

### Other information:


## Jetpack product discussion
pdDOJh-2jl-p2

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

1. Apply this PR to your local env
2. Spin up a Jurassic Tube site with this PR
3. Test with a site that has Newsletter Categories already defined
4. Add the Subscribe block to a post/page
5. Open the post/page and see if the categories are displayed
